### PR TITLE
Register new package ChebyshevApprox.jl v0.0.3

### PR DESCRIPTION
**Pull request by attobot**
Repository: [RJDennis/ChebyshevApprox.jl](https://github.com/RJDennis/ChebyshevApprox.jl)
Release: [v0.0.3](https://github.com/RJDennis/ChebyshevApprox.jl/releases/tag/v0.0.3)
cc: @RJDennis

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

@RJDennis This PR will remain open for 24 hours for feedback (which is optional). If you get feedback, please let us know if you are making changes, and we'll merge once you're done.

### CODE REVIEW

**Comment by tkelman** [comment]
Would Travis CI pass if you enabled it?

**Comment by RJDennis** [comment]
I've updated the require file and added tests for Travis.

**Comment by tkelman** [comment]
Doesn't look like you've actually turned Travis CI on, see https://travis-ci.org/RJDennis/ChevyshevApprox.jl

File: ChebyshevApprox/versions/0.0.3/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.5.0-rc1
```
Comment (tkelman) [inline ChebyshevApprox/versions/0.0.3/requires]: if you've changed this to 0.6 on master, should update the pr (see attobot readme) if you don't want to publish a version that supports 0.5

**Comment by aviks** [comment]
hi @RJDennis Do you want to merge this as is, or do you want to re-tag this for 0.6?

**Comment by MikeInnes** [comment]
Closing this as we haven't heard from you in a while. Do let me know if you want to re-open.

**Pull request closed without merging**